src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld.java

```diff
@@ -215,6 +216,14 @@ public boolean testForCubes(CubePos start, CubePos end, Predicate<Cube> cubeAllo
     @Override public Profiler getProfiler() {
         return this.theProfiler;
     }
+    
+    //Overwriting original World.markChunkDirty()
```

Comment:
This comment just repeats the code below it, seems completely unnecessary. Also Mixin AP (annotation processor) will complain if there is an overwrite without javadoc and if the javadoc has no `@reason` and `@author` tags.